pom.xml

```diff
@@ -90,14 +90,14 @@ under the License.
         <maven.version>2.2.1</maven.version>
         <maven-plugin-plugin.version>3.6.0</maven-plugin-plugin.version>
 
-        <scalac-scoverage-plugin.version>1.4.11</scalac-scoverage-plugin.version>
-        <scalac-scoverage-plugin.scala.version>2.12.15</scalac-scoverage-plugin.scala.version>
+        <scalac-scoverage-plugin.version>2.0.7</scalac-scoverage-plugin.version>
+        <scalac-scoverage-plugin.scala.version>2.13</scalac-scoverage-plugin.scala.version>

```

Comment:
I think this may have to be pinned to e.g. 2.13.10, to actually work? (going by available maven coordinates)
You may also want to bump the plugin version to 2.0.8 already

Reply:
It looks like this is only being used by the reporter down below, which doesn't need to be the exact version of Scala, only the binary 2.12, 2.13, or 3. The actual compiler plugin is what needs to be the full version.

Reply:
referencing that in the variable name would make it unmistakable.